File: app/src/main/java/marczak/pl/slowatrudne/NextObserver.java
package marczak.pl.slowatrudne;

import io.reactivex.Observer;
import io.reactivex.disposables.Disposable;

/**
 * Project "SlowaTrudne"
 * <p>
 * Created by Lukasz Marczak
 * on 05.03.2017.
 */

public abstract class NextObserver<T> implements Observer<T> {
    @Override
    public void onSubscribe(Disposable d) {

    }

    @Override
    public void onComplete() {

    }
}

File: app/src/main/java/marczak/pl/slowatrudne/utils/NextObserver.kt
package marczak.pl.slowatrudne.utils

import io.reactivex.Observer
import io.reactivex.disposables.Disposable

/**
 * Project "SlowaTrudne"
 *
 *
 * Created by Lukasz Marczak
 * on 05.03.2017.
 */

abstract class NextObserver<T> : Observer<T> {

    override fun onSubscribe(d: Disposable) {

    }

    override fun onComplete() {

    }
}
